aws.pkr.hcl

```diff
@@ -166,12 +171,14 @@ build {
   }
 
   post-processor "amazon-ami-management" {
-    # Deregister old AMIs, keep only the latest 180.
     regions = var.ami_regions
     tags = {
       Name = "Spacelift AMI"
     }
-    keep_releases = 180
+
+    # Deregister old AMIs, keep only the latest 180. If disabled, set
+    # to an absurdly high number to avoid deletion.
+    keep_releases = var.enable_image_cleanup ? 180 : 1000000
```

Comment:
Isn't it better to use the `only` feature ?

https://developer.hashicorp.com/packer/docs/templates/hcl_templates/blocks/build/post-processor#run-on-specific-builds

Reply:
I didn't know about this. Worth a try.